tests/backend_test/main.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+#. ../assert.sh continue_on_error #  load test functions with continue_on_error/stop_on_error
```

Comment:
Because this line is commented out, the tests fail to run.   Is the expectation that the user is running from the `backend_test` folder or the parent `test` folder?

Reply:
Parent test folder. executed from main.sh